build/templates/session.py.mako

```diff
@@ -143,6 +145,13 @@ class Session(object):
         self.library = library.get_library()
         self.vi = 0  # This must be set before calling _init_with_options.
         self.vi = self._init_with_options(resource_name, id_query, reset_device, options_string)
+
+        self.__isfrozen = True
```

Comment:
They are both needed. Comment added.

Without the class one, then __setattr__ raises an exception during __init__